e2e_testing/torchscript/elementwise.py

```diff
@@ -310,3 +310,36 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ElementwiseClampModule())
 def ElementwiseClampModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 5, low=-10, high=10))
+
+
+class RsubModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.rsub(x, 0.0, alpha=1.0)
+
+@register_test_case(module_factory=lambda: RsubModule())
+def RsubModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4))
+
+class RsubModule_noalpha(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.rsub(x, 0.0)
```

Comment:
nit: It might be better to use a different value other than `0.0` because zero can come from other places when they are initialized to default values. For example, In the bug that @cathyzhyi mentioned, the reason this was working on your machine was because the output tensor was being obtained from memory that had been initialized to all zeros, not because of the input being given, which can make things very confusing!